src/metabase/models/setting.clj

```diff
@@ -932,7 +932,7 @@
   (binding [*database-local-values* nil]
     (into
      {}
-     (comp (filter (fn [[_ setting]]
+     (comp (filter (fn [[_setting-name setting]]
```

Comment:
just an aside, I love the convention of `_setting-name` to make this easier to read. Thanks